consensus/metrics.go

```diff
@@ -50,101 +54,107 @@ type Metrics struct {
 }
 
 // PrometheusMetrics returns Metrics build using Prometheus client library.
-func PrometheusMetrics(namespace string) *Metrics {
+// Optionally, labels can be provided along with their values ("foo",
+// "fooValue").
+func PrometheusMetrics(namespace string, labelValues ...string) *Metrics {
+	labels := []string{}
+	for i := 0; i < len(labelValues)/2; i += 2 {
```

Comment:
I don't think this is right - we're looping over half the length and incrementing i by 2 each time ... this won't catch everything. We need to either loop over the whole length and increment by 2 each time, or loop over half and multiple the counter by 2 when indexing ...

Reply:
Duh :( Thanks for catching this